grove/measurements/term_grouping.py

```diff
@@ -0,0 +1,146 @@
+"""
+Various ways to group sets of Pauli Terms
+
+This augments the existing infrastructure in pyquil that finds commuting sets
+of PauliTerms.
+"""
+from pyquil.paulis import check_commutation
+
+
+def check_trivial_commutation(pauli_list, pauli_two):
+    """
+    Check if a PauliTerm trivially commutes with a list of other terms.
+
+    :param list pauli_list: A list of PauliTerm objects
+    :param PauliTerm pauli_two_term: A PauliTerm object
+    :returns: True if pauli_two object commutes with pauli_list, False otherwise
+    :rtype: bool
+    """
+    if not isinstance(pauli_list, list):
+        raise TypeError("pauli_list should be a list")
+
+    def _commutes(p1, p2):
```

Comment:
this doesn't need to be a closure. You can put it as a top level (private) function